### ISSUE TITLE
Agitator broken, needs updating

### ISSUE THREAD

**Issue by DomGarguilo** [apache/accumulo-testing#182]
It looks like the agitator is out of date and no longer works. 

Running `./bin/agitator start` results in the following output:
```
Using /home/dgarguilo/github/accumulo-testing/conf/env.sh for setup
Running master-agitator and tserver-agitator as dgarguilo
Running datanode-agitator as dgarguilo
```
Which seems to suggest that things are working. However, taking a look at the agitator.err logs, I see:
```
nohup: ignoring input
cat: /home/dgarguilo/github/fluo-uno/install/accumulo-2.1.0-SNAPSHOT/conf/tservers: No such file or directory
WARN setting maxKill to 0
WARN setting minKill to equal maxKill
```
Taking a quick look at the code, it seems there are several things that need to be updated (tserver location, master vs. manager, etc.) in order for agitator to work with the current state of accumulo. It might also be helpful to display to the user when something is awry instead of them needing to look into the logs before knowing there is an issue.

**Comment by milleruntime** [comment]
I can take a look at this. A work around for the tservers file would be to just create it once the cluster is setup. As a reference, it should still be around in the 1.10 distribution, under its former name "slaves".

**Comment by DomGarguilo** [comment]
Closed via #183
